jenkins/master/Dockerfile

```diff
@@ -11,16 +12,19 @@ RUN yum -y install openssl gnutls-utils \
 
 # fetch certificates and store them in tmp directory
 COPY ./import_certs.sh /usr/local/bin/import_certs.sh
-RUN import_certs.sh
+RUN chmod o+x /usr/local/bin/import_certs.sh \
```

Comment:
hi! have you tried `sh import_certs.sh` instead of having to change permission set?

Reply:
I don't have a preference, yet.
If permissions are not set as needed, those might be wrong any way.
This also means that `sh import_certs.sh` could execute successfully, but permissions of `import_certs.sh` might not be as expected.